docs/getting-started/migrate/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrate from the legacy Shopify App CLI
+section: getting-started
+redirect_from: "/migrate/"
```

Comment:
Is that URL something we used before?

Reply:
I wanted to use a redirect so that a) we can use a shorter URL inline in the CLI; and b) so we can always change the redirect destination later if we need to...